overview.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: index
+title: Overview
+permalink: /overview/
+---
+
+This page provides an overview of how Kroxylicious works.  For more details, please refer to the [documentation](./kroxylicious).
+
+#### What is Kroxylicious?
+
+Kroxylicious is an Apache Kafka&#174; protocol-aware proxy.  It can be used to layer uniform behaviours onto a
+Kafka based system in areas such as data-governance, security, policy enforcement and audit without needing to
+change either the applications or the Kafka Cluster.
+
+Kroxylicious is a stand-alone component which is deployed between the applications that use Kafka and
+the Kafka Cluster.  Instead of applications connecting straight to the Kafka Cluster, the applications connect to
+Kroxylious which connects to the cluster on the application's behalf. In all other respects, introducing 
+Kroxylicious into a Kafka system is transparant.
+
+To adopt Kroxylicious, there are zero code changes required to the applications. There are no additional libraries to
+install.  Kroxylicious supports applications written in any language supported by Kafka (Java, Golang, Python, Rust...).
+
+On the Kafka Cluster side, there are no changes required either.  That is, no additional libraries to deploy or special
+configurations to apply.  Kroxylicious works with both on-premise Kafka Clusters or cloud Kafka services.
```

Comment:
I feel like I want to add something about self managed -vs- provider managed Kafka services but I'm not sure how to add that into the on-premise -vs- cloud access.

Reply:
I've rephrased.. what do you think? @SamBarker

Reply:
Yes looks good 👍